Sources/Responses/EventContainer.swift

```diff
@@ -9,6 +9,7 @@
 import Foundation
 
 /// Container for Events
+@available(*, deprecated, message: "This class is obsoleted and will be removed in v5.0. Please use EntryContainer<Element> instead.")
```

Comment:
Can you briefly explain this one, please?

Reply:
`EventContainer` and `TrackingCode` classes are not used in our code and should be removed from repository.
The @available attribute used here will inform the consumer of this class by displaying a warning message in his code with the message placed in `message` field.